conference/debug_panel.py

```diff
@@ -87,3 +90,76 @@ def debug_panel_index(request):
     return TemplateResponse(request, "conference/debugpanel/index.html", {
         'debug_vars': debug_vars
     })
+
+
+# business functions
```

Comment:
I'd put those at the bottom of `conference/invoicing.py`

Reply:
done